### PR TITLE
Es6 datatable

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

**Comment by itkg-nanne** [comment]
:100:

File: BackofficeBundle/Resources/public/ecmascript/Lib/backboneHistory.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+            return '';
+        };
+
+        if ('undefined' !== typeof route) {
+            route = route.replace(optionalParam, replaceFunction);
+            route = route.replace(namedParam, replaceFunction);
+            return route
+        }
+
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/Lib/backboneHistory.js]: Can you check that the generated route is valid, I mean that all required parameter are given ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Collection/OrchestraCollection.js [hunk 2]
```diff
@@ -35,8 +35,8 @@ class OrchestraCollection extends Backbone.Collection
      */
     sync(method, model, options) {
         options = options || {};
-        let url = this._getSyncUrl[method.toLowerCase()];
-        if (typeof url !== 'undefined') {
+        let url = this._getSyncUrl()[method.toLowerCase()];
+        if(typeof url != 'undefined') {
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Collection/OrchestraCollection.js]: if (

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**